src/main/java/Exception/InvalidCommandException.java

```diff
@@ -0,0 +1,5 @@
+package Exception;
+
+public class InvalidCommandException extends Exception {
+
```

Comment:
Here, use the opportunity provided by inheritance to include a more expressive error message rather than just create an empty class